docker/yb-voyager-docker

```diff
@@ -15,9 +17,25 @@ fi
 
 i=0
 exported_vars=""
+volume_mappings=""
 
 # Define an array of environment variable names to capture
-variables=("BETA_FAST_DATA_EXPORT" "SOURCE_DB_PASSWORD" "TARGET_DB_PASSWORD" "FF_DB_PASSWORD" "YB_MASTER_PORT" "YB_TSERVER_PORT" "QUEUE_SEGMENT_MAX_BYTES" "NUM_EVENT_CHANNELS" "EVENT_CHANNEL_SIZE" "MAX_EVENTS_PER_BATCH" "MAX_INTERVAL_BETWEEN_BATCHES")
+variables=("BETA_FAST_DATA_EXPORT" 
+"SOURCE_DB_PASSWORD" 
+"TARGET_DB_PASSWORD" 
+"FF_DB_PASSWORD" 
```

Comment:
@ShivanshGahlot  Change FF_DB_PASSWORD -> SOURCE_REPLICA_DB_PASSWORD 
and add
`YB_VOYAGER_SEND_DIAGNOSTICS`